tools/snitch/snitcher/docker/linux/amd64/Dockerfile

```diff
@@ -1,21 +1,12 @@
-FROM ekidd/rust-musl-builder AS builder
-
-ARG SRC_DIR=.
-
-# Add source code
-COPY $SRC_DIR/ ./
-
-# Fix permissions on source code.
-RUN sudo chown -R rust:rust /home/rust
-
-# Build app
-RUN cargo build --release
-
-# Build real container
-
-FROM alpine:latest
-RUN apk --no-cache add ca-certificates
-COPY --from=builder /home/rust/src/target/x86_64-unknown-linux-musl/release/snitcher \
-    /usr/local/bin/
-
+FROM ubuntu:18.04
```

Comment:
Why do you need this change?

Reply:
It has error from 1 of build steps.  Let me run again and copy the error here.

Reply:
changed back to use 2 build stages.